packages/sdk/chainDeploy/helpers/liquidator.ts

```diff
@@ -32,3 +35,34 @@ export const deployFuseSafeLiquidator = async ({
   if (fsl.transactionHash) await ethers.provider.waitForTransaction(fsl.transactionHash);
   console.log("FuseSafeLiquidator: ", fsl.address);
 };
+
+export const configureFuseSafeLiquidator = async ({
+                                                 ethers,
+                                                 getNamedAccounts,
+                                                 chainId,
+                                               }: LiquidatorConfigFnParams): Promise<void> => {
+  const { deployer } = await getNamedAccounts();
+
+  const strategies: string[] = [];
+  const arrayOfTrue: boolean[] = [];
+  const fuseSafeLiquidator = (await ethers.getContract("FuseSafeLiquidator", deployer)) as FuseSafeLiquidator;
+
+  for (const address in chainRedemptionStrategies[chainId]) {
+    const redemptionStrategyType = chainRedemptionStrategies[chainId][address];
+    const redemptionStrategy = await ethers.getContract(redemptionStrategyType, deployer);
```

Comment:
@carlomazzaferro check the fix here for whitelisting of the actual redemption strategies